blocks/alternative_mutators/plus_minus/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Webpack configuration file.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = env => {
```

Comment:
@samelhusseini to look into making a root `webpack.config.js`, similar to the one for plugins.

Reply:
Should this be published? If so, then we would want it under packages/ in which case it would share the ``webpack.config.js`` of all other packages.